### PR TITLE
Add Support and Security metadata pages

### PR DESCRIPTION
**Pull request by oleg-nenashev**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Reporting Security Issues
+
+You can submit any security issue or suspected vulnerability
+on the GitHub Security pages of the project.
```
Comment (martinbonnin) [inline SECURITY.md:6]: I feel like I'm missing something here. I would expect `security@gradleup.com` here. Or is there a way to have some kind of "private issues" in GitHub for security?

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Support for GradleUp projects
+
+Any bug reports, feature requests and comments are welcome!
+The project keeps evolving,
+and any feedback from end users and developers will be appreciated.
+At the moment, there is no public Slack channel,
```
Comment (martinbonnin) [inline SUPPORT.md]: Nit: no need to mention it if it's not there. This way this paragraph will still be true when we add the channel.

```suggestion
and any feedback from end users and developers will be appreciated.
```

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+which are available in project repositories.
+Read More - [Security Policy](./SECURITY.md).
+
+## Setting Expectations
+
+GradleUp is a community driven project,
+most of contributors and maintainers invest their personal time.
+We do not guarantee a response or resolution
+time for the submitted issues.
+As it is written in the Apache License v2:
```
Comment (martinbonnin) [inline SUPPORT.md:26]: 👍

File: SUPPORT.md (hunk 2 above)
Comment (cortinico) [reply SUPPORT.md]: Agree on this 👍

File: SECURITY.md (hunk 1 above)
Comment (cortinico) [reply SECURITY.md:6]: Turns out we can turn private issue reporting for vulnerabilites
https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository

**Review by cortinico — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (martinbonnin) [reply SECURITY.md:6]: Nice! TIL.

Comment (oleg-nenashev) [reply SECURITY.md:6]: It is something we should add to the project onboarding checklist IMHO.
But yeah, it becomes a useful thing for small projects

**Comment by oleg-nenashev** [comment]
Merging to kick-off the site

File: SECURITY.md (hunk 1 above)
Comment (oleg-nenashev) [reply SECURITY.md:6]: Raised https://github.com/GradleUp/gradleup.github.io/issues/2 for now

**Pull request merged**